### PR TITLE
Dev docs

### PR DESCRIPTION
**Pull request by daniaki**
Add new docs to help developers get setup locally.

### CODE REVIEW

**Review by violetbrina — APPROVED** [review]
Perfect! I don't think that there is anything left out.

File: DEVELOPERS.md
Comment (jmarshall) [inline DEVELOPERS.md:9]:
```suggestion
Since most of our org use Apple laptops for development, this rest of this guide assumes
```

File: DEVELOPERS.md
Comment (jmarshall) [inline DEVELOPERS.md:14]:
```suggestion
Set up homebrew according to the instructions [here](https://brew.sh/). Before closing
```

**Review by jmarshall — COMMENTED** [review]
Typo patrol, but otherwise looks very helpful.

File: DEVELOPERS.md
Comment (daniaki) [reply DEVELOPERS.md:14]: Thanks! Will likely wait until the hail fix to merge this.

**Pull request merged**
